Conceptpower+Spring/src/main/webapp/WEB-INF/web.xml

```diff
@@ -60,5 +60,10 @@
 		<servlet-name>appServlet</servlet-name>
 		<url-pattern>/</url-pattern>
 	</servlet-mapping>
+    
+    <!--  Added session timeout as one hour -->
```

Comment:
comment needs to be corrected. for next time, the branch should have been called 'hotfix' to distinguish from regular bugs.